### ISSUE TITLE
fatal error while using the make command

### ISSUE THREAD

**Issue by ArekkusuMagni** [Dink4n/canime#4]
src/web_client.c:3:10: fatal error: curl/curl.h: No such file or directory
    3 | #include <curl/curl.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:28: obj/web_client.o] Error 1

...this is the error I get when I input "sudo make clean install"

**Comment by Dink4n** [comment]
Duplicate of #3 
If you are using ubuntu install `libcurl4-openssl-dev`
```
sudo apt install libcurl4-openssl-dev
```

scraping doesn't work in `canime` at the moment. I am working on fixing it

**Comment by ArekkusuMagni** [comment]
thank you that allowed me to use the make command. I am using MX Linux, based on Debian Bullseye. 

Now that canime is installed when I try to search for anime it tells me that it can't find it. 

Sabanto@X201MX:~
$ canime
Enter Anime: ghost in the shell
Couldn't find ghost in the shell
. Use a better search term.
Sabanto@X201MX:~
$ canime onepiece
Couldn't find onepiece. Use a better search term.
Sabanto@X201MX:~
$ canime
Enter Anime: serial experiments lain
Couldn't find serial experiments lain
. Use a better search term.

**Comment by Dink4n** [comment]
Yeah, scraping doesn't work. It's hard to manage `ani-cli` and `canime`.

I'll fix it now
